Contest_API.md

```diff
@@ -1076,44 +1076,96 @@ Returned data:
 ]
 ```
 
-### Team members
+### People
 
-Team members of teams in the contest.
+People involved in the contest.
 
-The following endpoints are associated with languages:
+The following endpoints are associated with people:
 
-| Endpoint                           | Mime-type        | Required? | Description
-| :--------------------------------- | :--------------- | :-------- | :----------
-| `/contests/<id>/team-members`      | application/json | no        | JSON array of all team members with elements as defined in the table below.
-| `/contests/<id>/team-members/<id>` | application/json | no        | JSON object of a single team member with elements as defined in the table below.
+| Endpoint                     | Mime-type        | Required? | Description
+| :--------------------------- | :--------------- | :-------- | :----------
+| `/contests/<id>/people`      | application/json | no        | JSON array of all people with elements as defined in the table below.
+| `/contests/<id>/people/<id>` | application/json | no        | JSON object of a single person with elements as defined in the table below.
 
-JSON elements of team member objects:
+JSON elements of people objects:
 
 | Name        | Type           | Required? | Nullable? | Description
 | :---------- | :------------- | :-------- | :-------- | :----------
-| id          | ID             | yes       | no        | Identifier of the team-member.
+| id          | ID             | yes       | no        | Identifier of the person.
 | icpc\_id    | string         | no        | yes       | External identifier from ICPC CMS.
-| team\_id    | ID             | yes       | no        | [Team](#teams) of this team member.
-| name        | string         | yes       | no        | Name of team member.
-| email       | string         | no        | yes       | Email of team member.
+| team\_id    | ID             | no        | yes       | [Team](#teams) of this person. Required iff role is `team`.
+| name        | string         | yes       | no        | Name of the person.
+| title       | string         | no        | yes       | Title of the person, e.g. "Technical director".
+| email       | string         | no        | yes       | Email of the person.
 | sex         | string         | no        | yes       | Either `male` or `female`, or possibly `null`.
-| role        | string         | yes       | no        | One of `contestant` or `coach`.
-| photo       | array of IMAGE | no        | yes       | Registration photo of the team member.
+| role        | string         | yes       | no        | One of `contestant`, `coach`, or `staff`.
```

Comment:
Similar to account type, maybe this should be allowed to be more open ended? Let's think about it for a possible future change?